src/pages/APIs/spec.md

```diff
@@ -0,0 +1,270 @@
+| Replicas | Number of replica nodes in the cluster | Yes | 3 |
+| Resources | Resource specifications for instance | No | See doc below |
+| Tenant Header | Unique header for multi-tenancy | No | None |
+| Version | Annotator for Clinical Data Container Edition version | Yes | None |
+
+<AnchorLinks small>
+  <AnchorLink>Annotators</AnchorLink>
+  <AnchorLink>Configuration Storage</AnchorLink>
+  <AnchorLink>License</AnchorLink>
+  <AnchorLink>Network Policy</AnchorLink>
+  <AnchorLink>Replicas</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Tenant Header</AnchorLink>
+  <AnchorLink>Version</AnchorLink>
+</AnchorLinks>
+
+### `Annotators`
+
+#### Description
+
+The `Annotators` property tells the deployment which annotator pods should be created as part of the deployment.  At least one annotator need to be enabled and by default all annotators are enabled.
+
+- `Advanced Care Insights` : discovers contextual information for problems, procedures, and medications identified in unstructured text
+- `Attribute Detection` : discovers domain specific attributes and associated values to be discovered in unstructured clinical text
+- `Concept Detection` : discovers matching terms from concept dictionaries
+- `Concept Disambiguation` : ranks term revelency based on sentence and document level information
+- `Concept Value Detection` : used with concept detection to discover values for matched dictionary concepts
+- `Hypothetical Detection` : discovers spans of text that are the object of a hypothetical statement
+- `Model Broker`: discovers key clinical concepts, such as medications, diagnoses, and procedures, with contextual scores.
+- `Negation Detection` : discovers  the spans of text that are the object of a negation and also identifies the text that triggered the negation
+- `Ontology` : allows user to manage custom ontologies / dictionaries
+- `Spell Checker` : discovers misspelled words and phrases in a document and suggests corrections
+
+#### Example
+
+```yaml
+  Annotators:
+    Advanced Care Insights:
+      Enabled:  true
+    Attribute Detection:
+      Enabled:  true
+    Concept Detection:
+      Enabled:  true
+    Concept Disambiguation:
+      Enabled:  true
+    Concept Value Detection:
+      Enabled:  true
+    Hypothetical Detection:
+      Enabled:  true
+    Model Broker:
+      Enabled:  true
+    Negation Detection:
+      Enabled:  true
+    Ontology:
+      Enabled:  true
+    Spell Checker:
+      Enabled:  true
+```
+
+### `Configuration Storage`
+
+#### Description
+
+The `Configuration Storage` property tells the deployment whether persistent storage is enabled and the type of storage being used.  There are two required fields to configuration storage.
+
+- `Storage` : either File or S3 compatible storage.  
+
+- `File Storage` : whether persistent storage is enabled, default is true.  If persistent is enabled the following additional properties are available:
```

Comment:
Add a new line before the "If persistent ... " so it reads similar to the "If S3 compatible ..." sentence. Also, change the sentence to be "If persistent file storage is enabled, the following ..."